docs/adr/0002-deprecation-strategy.md

```diff
@@ -0,0 +1,52 @@
+# Deprecation strategy
+
+* Date: 2020-11-05
+
+Technical Story: https://github.com/theupdateframework/tuf/issues/1127
+
+## Context and Problem Statement
+
+We plan to refactor the reference implementation significantly and, as part of
+that effort, drop support for no-longer maintained versions of Python
+(see ADR 0002).
```

Comment:
I think this should be a reference to ADR 0001

Reply:
Absolutely right, good catch!